pkg/kedge/common/common.go

```diff
@@ -89,3 +87,14 @@ func PortAllowed(port int, portRule *pb.Adhoc_Port) bool {
 	}
 	return false
 }
+
+func ResolveHost(hostStr string) (string, error) {
+	addrs, err := DefaultALookup(hostStr)
+	if err != nil {
+		return "", err
+	}
+	if len(addrs) == 0 {
+		return "", err
```

Comment:
mhmhm that would trigger panics too potentially. Let's error here properly (err will be always nil here)